code/dce-downloader/python/README.MD

```diff
@@ -1,88 +1,259 @@
-# DCE Tools in Python
+# DCE Downloader Tools in Python
 
-Two Python scripts files are provided to get data from DCE service in BV. Please refer to https://developer.bazaarvoice.com/docs/displayable_content_export/ for details.
+Two Python scripts files are provided to get data from the Bazaarvoice Displayable Content Export service. Please refer to [documentation](https://developer.bazaarvoice.com/displayable-content-export) for details.
 
 ## Prerequisite
 1. You should have Python v2.7.10 or later installed.
-2. You should have requests installed (run "pip install requests")
-3. You should have a file called "keys.json" under "./dce-downloader/keys.json", which includes code snippet below.
-   - Please replace "x-api-key" and "shared secret" with real x-api-key key and shared secret key in keys.json before running scripts .
-    ```
-    {
-    "stg":
-        {
-            "x-api-key": "x-api-key",
-            "secret": "shared secret"
-        },
-    "prod":
-        {
-            "x-api-key": "x-api-key",
-            "secret": "shared secret"
-        }
-    }
-    ```
+2. You should have requests installed
```

Comment:
Since we've got a `requirements.txt`, we can just instruct them to `pip install` everything from there (rather than requests specifically). That would also enforce the version requirements in the `requirements.txt`. This is consistent with the instructions below (on line 53).

So something like:
```
2. You should install the requisite packages in `requirements.txt`
```